### PR TITLE
done first try

### PR DESCRIPTION
**Pull request by VladyslavKucher**
https://VladyslavKucher.github.io/layout_moyo-header/
https://VladyslavKucher.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,55 @@
...
+      <nav class="box text active">
+        Tablets
+      </nav>
+      <nav class="box text active">
+        Gadgets
+      </nav>
+      <nav 
+      class="box text active"
+      data-qa="hover"
+      >
```
Comment (qoa11a) [inline src/index.html:35]:
```suggestion
      <nav 
        class="box text active"
        data-qa="hover"
      >
```
fix code style everywhere

File: src/style.css [hunk 2]
```diff
@@ -1 +1,89 @@
...
+  top: 18.5px;
+  display: block;
+  border-bottom: #00acdc solid 4px;
+  border-radius: 8px;
+  opacity: 0;
+}
+
+.active:active::after {
+  opacity: 1;
+}
```
Comment (qoa11a) [inline src/style.css:75]: you shouldn't style only link with active class. Every link should have styles on hover but not on click in this task. Class active is only for tests

**Review by qoa11a — CHANGES_REQUESTED** [review]
don't name classes using camelCase
don't forget to use required font-family

File: src/style.css [hunk 3]
```diff
@@ -1 +1,84 @@
...
+
+.box {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-right: 20px;
+  display: inline-block;
+  font-family: "Roboto", sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/style.css:21]: move it to the html selector

File: src/style.css [hunk 4]
```diff
@@ -1 +1,84 @@
...
+  top: 50%;
+  transform: translateY(-50%);
+  display: flex;
+  flex-grow: 1;
+  margin-left: 50px;
+}
+
+.box:hover {
+  cursor: pointer;
+  color: #00acdc;
```
Comment (tetiana-ivanchenko) [inline src/style.css:54]: move repeatable colors in the variables

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**